tests/v2/actions/workloads/pods/pods.go

```diff
@@ -137,28 +134,19 @@ func WatchAndWaitPodContainerRunning(client *rancher.Client, clusterID, namespac
 
 	namespacedClient := steveclient.SteveType(podSteveType).NamespacedSteveClient(namespaceName)
 
-	dynamicClient, err := client.GetDownStreamClusterClient(clusterID)
-	if err != nil {
-		return err
-	}
-
-	deploymentResource := dynamicClient.Resource(deployments.DeploymentGroupVersionResource).Namespace(namespaceName)
-
-	watchAppInterface, err := deploymentResource.Watch(context.TODO(), metav1.ListOptions{
-		FieldSelector:  "metadata.name=" + deploymentTemplate.Name,
-		TimeoutSeconds: &defaults.WatchTimeoutSeconds,
-	})
-	if err != nil {
-		return err
+	backoff := kwait.Backoff{
+		Duration: 10 * time.Second,
```

Comment:
based on our discussion I would reduce this time to 1

Reply:
done

Reply:
@slickwarren I've tested and 1 second is few time, I am getting timeout error now, could be 5 seconds? it working fine